src/esp/physics/bullet/BulletRigidObject.cpp

```diff
@@ -281,16 +285,18 @@ bool BulletRigidObject::setMotionType(MotionType mt) {
         ~btCollisionObject::CF_KINEMATIC_OBJECT);
     objectMotionType_ = MotionType::STATIC;
 
-    // create a static collision object at the current transform
-    std::unique_ptr<btCollisionObject> staticCollisionObject =
-        std::make_unique<btCollisionObject>();
-    staticCollisionObject->setCollisionShape(bObjectShape_.get());
-    staticCollisionObject->setWorldTransform(
-        bObjectRigidBody_->getWorldTransform());
-    bWorld_->addCollisionObject(
-        staticCollisionObject.get(),
-        2,       // collisionFilterGroup (2 == StaticFilter)
-        1 + 2);  // collisionFilterMask (1 == DefaultFilter, 2==StaticFilter)
+    // mass == 0 to indicate static. See isStaticObject assert below. See also
```

Comment:
This new code is mostly copied from https://github.com/facebookresearch/habitat-sim/pull/814, except notice also the new collision filtering.